### ISSUE TITLE
sass-graph should prefer cwd over loadPaths for @imports

### ISSUE THREAD

**Issue by schnerd** [xzyfer/sass-graph#12]
I ran into an issue running sass-graph on a larger project. There are two modules within this project that contain a "_base.scss". All scss files in module 1 that imported _base worked correctly, but when it was time to compile scss files in module 2, their @import statements were resolving the _base.scss from module 1 instead of the one in the CWD.

```
mod1/_a.scss
mod1/b.scss
   @import "a";
mod2/_a.scss
mod2/b.scss    
   @import "a";  // graph.index['mod2/b.scss'].imports === ['mod1/_a.scss']
```

By default, I believe Sass will look for imports in the CWD and then look in any include paths from there
> Sass looks for other Sass files in the current directory, and the Sass file directory under Rack, Rails, or Merb. Additional search directories may be specified using the :load_paths option, or the --load-path option on the command line. - [Source](http://sass-lang.com/documentation/file.SASS_REFERENCE.html#import)

I'm opening up a PR that I think will fix the issue, curious to hear your feedback.

**Comment by xzyfer** [comment]
Sorry I haven't been receiving GH emails.

Makes sense! I'll take a look shortly. Thanks!

**Comment by danbroooks** [comment]
+1 for this

**Comment by jfridye** [comment]
+1

**Comment by kkopachev** [comment]
+1. Could not use node-sass watcher because of this, it's always compiling wrong partials.

**Comment by xzyfer** [comment]
My apologies for the silence here. This issue is top priority and will be resolved shortly for the node-sass@3.0.0 release. https://github.com/sass/node-sass/issues/905
